### PR TITLE
[SD-3448] - Related Items and Duplicates separation

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: client/app/scripts/superdesk-archive/module.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
     'angular',
     'require',
+    'moment',
     './controllers/list',
     './controllers/upload',
     './archive-widget/archive',
     './related-item-widget/relatedItem',
     './directives'
-], function(angular, require) {
+], function(angular, require, moment) {
```
Comment (sivakuna-aap) [inline client/app/scripts/superdesk-archive/module.js]: Is moment required? There' no usage of it?

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**